code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -181,6 +181,16 @@
 	..()
 	reagents.add_reagent(INSECTICIDE, 250)
 
+//Fake Xeno Creep Sprayer
+/obj/item/weapon/reagent_containers/spray/creepspray
+	name = "Creep Spray"
```

Comment:
I don't think this name is very descriptive for a player seeing it in the autodrobe